image/share/Taskfile

```diff
@@ -0,0 +1,3 @@
+default: KMER=121 PARTITIONS=4
+fast: KMER=63 PARTITIONS=128
+faster: KMER=31 PARTITIONS=256
```

Comment:
@michaelbarton Thanks for refactoring. Concerning the poor assembly: I also noticed it, but the assembler is being called correctly to my knowledge. We need to check how it performs with other samples.